pyproj/_crs.pyx

```diff
@@ -2834,15 +2834,19 @@ cdef class _CRS(Base):
         # get list of possible matching projections
         cdef PJ_OBJ_LIST *proj_list = NULL
         cdef int *c_out_confidence_list = NULL
-        cdef int num_proj_objects = -9999
+        cdef int num_proj_objects = 0
         cdef bytes b_auth_name
         cdef char *user_auth_name = NULL
         cdef int iii = 0
+        cdef PJ * proj = NULL
```

Comment:
Please don't change the location of the cdef for these. It is preferred to define these closer to where they are going to be used.

Reply:
Pushed a new change that is more minimal and does not touch this.